### PR TITLE
[LPS-148755 & LPS-148756 & LPS-149105] [CP 2.0] Team Members remove / edit roles

### PR DESCRIPTION
**Pull request by iagochaves**

### CODE REVIEW

**Review by jairmedeiros — APPROVED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/constants/slaTypes.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+const SLA_TYPES = {
+	gold: 'Gold',
+	platinum: 'Platinum',
+};
```
Comment (igordmf) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/constants/slaTypes.js:15]: We have a constant like this inside InviteTeamMembersPage component. Could you change there to use this SLA_TYPES?

Almost same for SlaCardLayout component, but for that have one more key-value pair, so if it makes senses, add `limited:"Limited"` on the SLA_TYPES and use there. I don`t know if it`s necessary, just saying,.

**Review by igordmf — APPROVED** [review]
Good job!

**Review by claricebrasil — APPROVED** [review]
Amazing, Iago! Congratulations

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/constants/slaTypes.js (hunk 1 above)
Comment (iagochaves) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/constants/slaTypes.js:15]: Hey @igordmf. About the SLA_TYPES, I've already changed for team members (https://github.com/pt-liferay-solutions/liferay-portal/pull/1052/files#diff-ff0b2b8535eecfcc0e7207a50d438e37faf4d5b90f9fb85904f11a9d1f2ea496R69)

Comment (iagochaves) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/constants/slaTypes.js:15]: Nice catch about SlaCardLayout, I'll be adding that

Comment (iagochaves) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/utils/constants/slaTypes.js]: Done

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by iagochaves** [comment]
ci:forward

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by iagochaves** [comment]
ci:forward

**Comment by iagochaves** [comment]
ci:forward

**Comment by iagochaves** [comment]
ci:forward

**Comment by iagochaves** [comment]
ci:forward

**Pull request closed without merging**